### DISCUSSION TITLE
Integration Search - Aliases / Tags

### DISCUSSION THREAD

**Discussion by ArmoredCavalry** [home-assistant/frontend#12328] [Dashboard] [answered]
Recently ran into a situation where a certain integration (motion-blinds) works with quite a few other brands that all use the same underlying software. For my particular brand, doing a search of integrations came up with no results in the UI. Occurred to me this could be something that happens with other integrations, where they have other "white-labeled" products, or abbreviations, or re-brands.

Speaking with @[starkillerOG](https://github.com/starkillerOG) on the motion-blinds repo, he mentioned proposing a feature for this. Perhaps an integration could specify "aliases" that show up in search, but link to the same underlying integration / documentation? Or, if not a separate entry / result, maybe integrations could at least add "tags", which will be searched on in addition to title? Then if you hover / long-press a given integration in search results, it shows the tags?

I think this could help with discoverability of integrations, and keep people inside the UI longer (without resorting to Google).

**Comment by frenck** [comment, marked as answer]
This is in progress (and not a frontend feature request).

You can follow the architectural discussion/progress here: <https://github.com/home-assistant/architecture/discussions/734>

**Reply by starkillerOG** [reply]
This PR https://github.com/home-assistant/core/pull/70996 inplements this for the Motion Blinds integration and is already merged.
